my-dsl.md

```diff
@@ -2,37 +2,78 @@
 _What is the name of the language? Link the name to its webpage 
 (if appropriate)._
 
+I looked at the language [Cg][1].
+
+[1]: https://developer.nvidia.com/cg-toolkit
+
 
 # Domain
 _Describe the language's domain in five words._
 
+vertex/pixel shading for GPUs
+
 
 # Computational model
 _What is the underlying computational model of this language? To answer this 
 question, provide a high-level description (no more than 100 words) of the 
 computation that occurs when someone executes a program in this language._
 
+According to the [Cg tutorial][2], section 1.4.2, Cg is executed to interact
+with the OpenGL or Direct3D API for your GPU. The way this works is for a Cg
+program to be translated into OpenGL or Direct3D, then for the translated
```

Comment:
This was a super hard question for me too, but I think that your answer is a little repetitive for what you were trying to say. The meat of what you said was __ Cg program to be translated into OpenGL or Direct3D__. Basically the meaning could have been conveyed using something like this:
_According to the [Cg tutorial](http://http.developer.nvidia.com/CgTutorial/cg_tutorial_chapter01.html) Cg works by being translated into your GPU API of choice and then being executed as such._